apps/comps/types/agent.ts

```diff
@@ -148,8 +153,7 @@ export interface CreateAgentRequest {
 }
 
 export interface CreateAgentResponse {
-  agent: Agent & { apiKey: string };
-  success: boolean;
+  agent: Omit<Agent, "stats" | "skills" | "trophies" | "hasUnclaimedRewards">;
```

Comment:
We'll have to move to just using the types returned by the router asap. It sucks to maintain these types and not mess something up.